### DISCUSSION TITLE
Save `jira issue list` to CSV

### DISCUSSION THREAD

**Discussion by luv-deluxe** [ankitpokhrel/jira-cli#678] [Ideas]
Hey.

Saving issues list to CSV would help to:
- synchronize it with Excel for further processing,
- synchronize it with UML tools like MagicDraw, where one can get JIRA Stories and refine them to Use Cases.
- more to come...

I've saved command output to file, but delimiter is hard to handle. CLI output puts multiple tabs to align columns, thus importing it to Excel is problematic, as column values are put in wrong places.
One can select `consecutive delimiters` option, but then, if e.g. ASSIGNEE column is empty, it's printed as `\t\t`, not as `\t""\t`. This also shifts values to the left, creating confusing Excel.

Maybe, there could be a switch to `issue list` command, to not pretty-print table, but use a custom separator. This would enable custom processing.

**Comment by ankitpokhrel** [comment]
I think this is a fair ask. A `delimiter` flag that works with `--plain` won't hurt IMO.

**Reply by luv-deluxe** [reply]
Sure. Setting e.g. `;` as column delimter can would render `;;` in presence of empty cell. Thus, it can be dumped to file with a simple `>` operator and then used according to own needs.

`jira issue list --plain --delimiter=; > issues.txt`

**Reply by danpritts** [reply]
for anyone coming through later, note that `;` in particular is tricky; it is typically a command separator in *nix shells.  safest to use `';'` or `\;`

**Comment by ElCaptaine** [comment]
if you want to append the issues to an existing csv use `>>` instead of `>`

**Comment by ankitpokhrel** [comment]
The upcoming release will include `--raw` for JSON output and `--csv` for CSV output. I believe csv option addresses this concern.
